### DISCUSSION TITLE
landscape mode

### DISCUSSION THREAD

**Discussion by el-lolo** [Jyers/Marlin#770] [Q&A] [answered]
hi 
I  would like to know if there is a way to have the screen en landscape mode . 
In configuration.h  "LCD_INFO_SCREEN_STYLE" classic or prusa is the way to config it?

**Comment by Jyers** [comment, marked as answer]
This is in progress, but will take a lot of work to implement. So it might be a but before it will be available.

**Reply by el-lolo** [reply]
Thanks a lot . I found  this tuto there is the landscape mode . (you have a marlin folder ) . I follow it for no bltouch version and it  work . I hope that can give you a hand

**Reply by el-lolo** [reply]
https://otioss.com/posts/ender3-v2-front-panel

**Reply by Jyers** [reply]
Yes, I have talked with him about implementing his layout as an option. Hopefully we can get it done

**Reply by DampferDealz** [reply]
Iam looking forward for landscape modus so much, it would be a great thing !!

**Reply by alexqzd** [reply]
> Yes, I have talked with him about implementing his layout as an option. Hopefully we can get it done

That layout is nice. It would be great if you could fit the Creality logo in there. Maybe putting XYZ coordinates in the same row, and temps in the same row too

**Reply by Gil80** [reply]
Landscape mode FTW! :)
Please have this implemented. It would be simply the best of Jyers and Landscape for the E3V2!! 💯
